### ISSUE TITLE
font size not customizable

### ISSUE THREAD

**Issue by jonathan-richer** [michael-brade/LaTeX.js#102]
I can't change the font size of the TeX, I'm using the HTML <latex-js> component and it gets overidden by `--size: 13.284px;` whatever I do.

**Comment by michael-brade** [comment]
Yes, you are right. I remember there was a limitation when using webcomponents, but I cannot quite remember. I will do some research again and then document it here in this issue so that I won't forget again and maybe someone knows a workaround. But I don't have much time at the moment, so it will take a few more days.
